controllers/remote/cluster_cache.go

```diff
@@ -51,21 +52,51 @@ const (
 
 // ClusterCacheTracker manages client caches for workload clusters.
 type ClusterCacheTracker struct {
-	log    logr.Logger
-	client client.Client
-	scheme *runtime.Scheme
+	log                   logr.Logger
+	clientDisableCacheFor []client.Object
```

Comment:
May be name this `clientUncachedObjects` or just `uncachedObjects (since this is a field of `ClusterCacheTracker` ) to be consistent `NewDelegatingClientInput`